jenkins/github_helpers.py

```diff
@@ -201,22 +201,28 @@ def create_pull_request(self, repository, title, body, base, head, user_reviewer
 
     def verify_reviewers_tagged(self, pull_request, requested_users, requested_teams):
         """
-        Assert if the reviewers we requested were tagged on the PR for review
+        Assert if the reviewers we requested were tagged on the PR for review.
+
+        Considerations:
+
+        - Teams cannot be tagged on a PR unless that team has explicit write
+          access to the repo.
+        - Github may have independently tagged additional users or teams
+          based on a CODEOWNERS file.
         """
         # TODO: Remove is_cleanup_pr in favor of a new CLI option on
         # PullRequestCreator that makes tagging failures log-only
         if self.is_cleanup_pr(pull_request.head.ref):
             return
         tagged_for_review = pull_request.get_review_requests()
-        tagged_users = [user.login for user in tagged_for_review[0]]
-        tagged_teams = [team.name for team in tagged_for_review[1]]
-        logger.info("Actually tagged on PR: users=%s teams=%s", tagged_users, tagged_teams)
 
-        if requested_users is not GithubObject.NotSet and sorted(requested_users) != sorted(tagged_users):
+        tagged_users = [user.login for user in tagged_for_review[0]]
+        if not (requested_users is GithubObject.NotSet or set(requested_users) <= set(tagged_users)):
```

Comment:
I've inverted the logic here to state the positive case more clearly -- the set logic was harder to read with a not floating out front of the comparison.